sdk/python/docs/JobsetV1alpha2PodAnnotator.md

```diff
@@ -0,0 +1,12 @@
+# JobsetV1alpha2PodAnnotator
```

Comment:
Looks like something is up with our sdk generation.  I'd expect to see some python code in this PR.

Reply:
Come to think of it, I don't see any client-go changes either.